website/docs/docs/dbt-versions/dbt-cloud-release-notes-gen.md

```diff
@@ -17,6 +17,58 @@ unlisted: true
+
+- **Insights UI**
+  - **Copilot guardrails**: The Copilot listener now hydrates builder tabs only when a semantic-layer payload arrives, preventing plain-SQL replies from overwriting editor state. 
+- **dbt CLI**
+  - **Improved monorepo support for file sync and the IDE**:
+    - File sync now anchors itself to the invocation directory, making monorepo structures behave more predictably.
+    - Nested `dependencies.yml` files correctly trigger dependency installs.
+    - The IDE’s LSP and file sync now recognize dbt subdirectories properly.
+    - Exclusion lists remain accurate even in multi-project repositories.
+- **Notifications system**
+  - **Webhook auditability**: Outbound calls now persist the exact JSON body in webhook history, making allowlisting and troubleshooting easier. 
+
+- **Studio**
+  - **Git sidebar + file refresh parity**: The file tree now mirrors Cloud VCS statuses (including conflicts) and automatically invalidates caches after `dbt deps`/`dbt clean`, so new or removed files appear without a reload.
+  - **Log viewers & Autofix UX**: Command and interactive query logs adopt the new accordion-based viewer, and Autofix sessions in Fusion treat plain `parse` commands as the trigger for deprecation summaries, keeping remediation flows consistent.
+
+### Fixes
+
+- **dbt platform**
+  - **Environment variable editor stability**: Editing one variable no longer backfills blank cells with previously edited values, preventing accidental overrides. 
+  - **Cost optimization indicator accuracy**: Job pages once again display “Cost optimization features” whenever Fusion actually ran (and gating conditions are met), so customers see the right coverage status regardless of feature-flag permutations. 
+
+### Behavior changes
+
+- **dbt platform**
+  - **Stronger tenant identity enforcement**: Service/PAT calls without an active license now fail authentication, Slack Copilot sessions build a scoped identity JWT for the invoking user, and SSO providers enforce auto-generated slugs (draft configs can’t be targeted), reducing misconfiguration risk. 
+
+- **dbt CLI**
+  - **User-isolated invocation history**: Every invocation lookup validates the caller’s user ID, preventing admins from accidentally reading another developer’s runs when multiple accounts share a CLI server. 
+- **IDE server**
+  - **Support-impersonation guardrails & short-lived show artifacts**: Operators who “hijack” a customer session can no longer run show/run/build/test commands, and `dbt show` artifacts expire after 15 minutes to protect sensitive data. <!-- https://github.com/dbt-labs/helm-releases/tree/main/combined-diffs/diff-ide-server-20248398200 -->
+  - **Enhanced security for support-assisted sessions:** Support impersonation sessions now restrict the execution of `show`, `run`, `build`, and `test` commands. Artifacts generated by `dbt show` are also short-lived and will automatically expire after 15 minutes to limit unintended data retention.
```

Comment:
This seems to repeat the information above but in more palatable language. Maybe we delete like 66 and keep this?

Reply:
Oh oops! I was rewriting the previous bullet and forgot to delete it!